ConwaysGameOfLife.Tests/LocationShould.cs

```diff
@@ -33,17 +33,29 @@ public void WhenXAndYAreSame_BeEqualOperator()
             Assert.True(result);
         }
 
-        [Fact]
-        public void WhenTwoLocationsInstantiated_BeEqual()
+        [Theory]
+        [InlineData(0, 0, 0, 0, true)]
+        [InlineData(0, 0, 1, 0, false)]
+        public void WhenTwoLocationsInstantiated_BeEqual(int x1, int y1, int x2, int y2, bool expected)

```

Comment:
probably should only have true cases here( and then Assert.True later)
```suggestion
        [InlineData(0, 0, 0, 0)]
        [InlineData(2, 2, 2, 2)]
        public void WhenTwoLocationsInstantiated_BeEqual(int x1, int y1, int x2, int y2)

```

Reply:
Agree with Keddy. Try to only test 1 thing per test, one reason being so that if something fails you can easily identify which one. I recommend using multiple asserts in 1 test only if they are related (e.g. when comparing the contents of 2 cells and needed to have 3 asserts to check that it's the same)